imjoy/imjoyPluginEngine.py

```diff
@@ -825,6 +825,9 @@ def connect(sid, environ):
                 logger.debug("message from %s", pid)
                 if kwargs["type"] == "initialized":
                     addPlugin(plugin_info, sid)
+                elif kwargs["type"] == "executeFailure":
+                    logger.info("Killing plugin %s due to exeuction failure.", pid)
+                    killPlugin(pid)
```

Comment:
I added that to make sure that we kill the plugin when an executeFailure happened.